tests/e2e/system_test/test_cr_resources_validation.py

```diff
@@ -23,9 +23,9 @@
 
 
 @magenta_squad
-@tier3
-@skipif_ocp_version("<4.13")
-@skipif_ocs_version("<4.13")
+@tier2
+@skipif_ocp_version("<4.15")
+@skipif_ocs_version("<4.15")
 class TestCRRsourcesValidation(ManageTest):
```

Comment:
We discussed already couple of months back, adding it as a review comment here. 

Below tests appears to be functionality tests. They are currently placed under system_test. I dont think system_test is the appropriate place for these tests. 
test_reclaim_space_job_editable
test_volume_replication_class_editable
test_volume_replication_cr_editable

And also you may have to rebase your PR to reflect the new test directory structure

Reply:
@PrasadDesala  

Thank you for your review. 
Following our previous correspondance I'm moving the test to tests/manage/pv_services/space_reclaim/ 
The squad was changed to grey_squad ( which I belong to).